src/app/components/form/AppForm.tsx

```diff
@@ -1,41 +1,41 @@
 "use client";
 
-import { useForm } from "react-hook-form";
+import * as yup from "yup";
 import { useEffect, useState } from "react";
-// components
+import { getInstitutionApplication } from "../../utils/institutions";
+import { USWDSForm } from "./Form";
 import {
   Button,
   ButtonGroup,
   Card,
   CardBody,
   GridContainer,
 } from "@trussworks/react-uswds";
-import { TextField, TextArea, Spinner } from "@/src/app/components";
-import NotFound from "@/src/app/not-found";
+import { Spinner } from "../utilities/Spinner";
+import NotFound from "../../not-found";
+import { TextField } from "./TextField";
 import Link from "next/link";
+import { saveApplication } from "../../utils/applications";
 import { useRouter } from "next/navigation";
-// utils
-import { getInstitutionApplication } from "@/src/app/utils/institutions";
-import { saveApplication } from "@/src/app/utils/applications";
+import { email, number, numberOptional, text } from "../../schemas/schemas";
+import { TextArea } from "./TextArea";
```

Comment:
nit: can combine some of the imports from the component index file
Spinner, TextArea, TextField

Reply:
and if you do end up doing that, would be nice to organize these imports with comments like elsewhere

Reply:
Fixed up :D